ui/page/governance/consensus_page.go

```diff
@@ -275,6 +282,73 @@ func (pg *ConsensusPage) Layout(gtx C) D {
 	)
 }
 
+func (pg *ConsensusPage) layoutMobile(gtx layout.Context) layout.Dimensions {
+	return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+		layout.Rigid(func(gtx C) D {
+			return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
+				layout.Rigid(func(gtx C) D {
+					return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
+						layout.Rigid(pg.Theme.Label(values.TextSize20, values.String(values.StrConsensusChange)).Layout), // Do we really need to display the title? nav is proposals already
+						layout.Rigid(pg.infoButton.Layout),
+					)
+				}),
+				layout.Flexed(1, func(gtx C) D {
+					return layout.E.Layout(gtx, func(gtx C) D {
+						return layout.Inset{Right: values.MarginPadding10, Top: values.MarginPadding5}.Layout(gtx, pg.layoutRedirectVoting)
+					})
+				}),
+			)
+		}),
+		layout.Flexed(1, func(gtx C) D {
+			return layout.Inset{Top: values.MarginPadding10}.Layout(gtx, func(gtx C) D {
+				return layout.Stack{}.Layout(gtx,
+					layout.Expanded(func(gtx C) D {
+						return layout.Inset{
+							Top: values.MarginPadding60,
+						}.Layout(gtx, pg.layoutContent)
+					}),
+					// layout.Expanded(func(gtx C) D {
+					// 	gtx.Constraints.Max.X = gtx.Dp(values.MarginPadding150)
+					// 	gtx.Constraints.Min.X = gtx.Constraints.Max.X
+
+					//TODO: temp removal till after V1
+					// card := pg.Theme.Card()
+					// card.Radius = decredmaterial.Radius(8)
+					// return card.Layout(gtx, func(gtx C) D {
+					// 	return layout.Inset{
+					// 		Left:   values.MarginPadding10,
+					// 		Right:  values.MarginPadding10,
+					// 		Top:    values.MarginPadding2,
+					// 		Bottom: values.MarginPadding2,
+					// 	}.Layout(gtx, pg.searchEditor.Layout)
+					// })
+					// }),
+					layout.Expanded(func(gtx C) D {
+						gtx.Constraints.Min.X = gtx.Constraints.Max.X
+						return layout.E.Layout(gtx, func(gtx C) D {
+							card := pg.Theme.Card()
+							card.Radius = decredmaterial.Radius(8)
+							return layout.Inset{Right: values.MarginPadding10}.Layout(gtx, func(gtx C) D {
+								return card.Layout(gtx, func(gtx C) D {
+									return layout.UniformInset(values.MarginPadding8).Layout(gtx, func(gtx C) D {
+										return pg.layoutSyncSection(gtx)
+									})
+								})
+							})
+						})
+					}),
+					layout.Expanded(func(gtx C) D {
+						return pg.orderDropDown.Layout(gtx, 55, true)
+					}),
+					layout.Expanded(func(gtx C) D {
+						return pg.walletDropDown.Layout(gtx, pg.orderDropDown.Width+51, true)
+					}),
+				)
+			})
+		}),
+	)
+}
```

Comment:
I dont see the difference between what you have here and on the desktop layout.. Kindly help clarify why we need this?

Actually lookign again i see the dropdown position values changed if that be the only case, we can have only on function that takes the dropdown position as a parameter and the values can be passed to it by the different layouts.

```
 func (pg *ConsensusPage) pageLayout(gtx layout.Context, orderPos int, walletPos int) layout.Dimensions {}
```

```
if pg.Load.GetCurrentAppWidth() <= gtx.Dp(values.StartMobileView) {
		return pg.pageLayout(gtx, val, val)
	}
   return pg.pageLayout(gtx, val, val)
```

Reply:
we should leave as is, further details here https://github.com/planetdecred/godcr/pull/971#discussion_r904149266

Reply:
Then i believe it wont be out of order to properly comments these codes for easier reviews